autogpt_platform/autogpt_libs/autogpt_libs/supabase_integration_credentials_store/types.py

```diff
@@ -29,20 +29,29 @@ class OAuth2Credentials(_BaseCredentials):
     scopes: list[str]
```

Comment:
Can we change this folder name to something much shorter, as it makes for a very long import string

Reply:
What would you name it?